posthog/hogql/functions/mapping.py

```diff
@@ -1073,6 +1073,7 @@ def compare_types(arg_types: list[ConstantType], sig_arg_types: tuple[ConstantTy
     "maxIntersectionsPositionIf": HogQLFunctionMeta("maxIntersectionsPositionIf", 3, 3, aggregate=True),
 }
 HOGQL_POSTHOG_FUNCTIONS: dict[str, HogQLFunctionMeta] = {
+    "stringOr": HogQLFunctionMeta("stringOr", 1, None),
```

Comment:
Just commenting here as good as anywhere.

`stringOr` sounds really weird to me. Is it truly only for strings? If so then I guess its fine and I assume the naming is coming from `logicalOr`? Something like `firstNonEmptyString()` is obviously quite verbose but `stringOr` didn't mean a whole lot to me at first - could be my lack of common programming terms here.

If it is only for strings then I'm not so sure about adding it as a standard shorthand `^^` (I can't even find that char on my german keyboard 😅 ).